### ISSUE TITLE
Uncaught TypeError: Cannot read property 'append' of undefined

### ISSUE THREAD

**Issue by resure** [mauricioszabo/atom-rails-i18n#1]
[Enter steps to reproduce below:]
1. Trying to run "find key comand in the project without i18n keys

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.3
**Thrown From**: [rails-i18n](https://github.com/mauricioszabo/atom-rails-i18n) package, v0.1.2
### Stack Trace

Uncaught TypeError: Cannot read property 'append' of undefined

```
At /Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:19

TypeError: Cannot read property 'append' of undefined
  at Finder.initialize (/Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:19:23)
  at Finder.View (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at Finder.SelectListView (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:18:51)
  at new Finder (/Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:10:3)
  at atom-workspace.<anonymous> (/Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:44:11)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:180:20)

```
### Commands

```
     -0:19.4.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:19.3.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:19.3.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:19.2.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:19.1.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:18.9.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:18.8.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:18.7.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:18.6.0 vim-mode:move-up (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:18.5.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:18.1.0 pane:show-previous-item (atom-text-editor.editor.vim-mode.is-focused.command-mode)
     -0:08.1.0 command-palette:toggle (atom-text-editor.editor.vim-mode.command-mode.is-focused)
     -0:05.9.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -0:04.9.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:01.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:01.8.0 rails-i18n:search-key (atom-text-editor.editor.vim-mode.command-mode)
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "bemhtml",
      "language-yate"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
Stylus, v1.0.0
atom-typescript, v4.6.9
linter, v1.0.8
rails-i18n, v0.1.2
vim-mode, v0.54.0

# Dev
No dev packages
```

**Comment by mauricioszabo** [comment]
I use it mostly with the "Everything" plugin, but I'll correct this bug (deprecated API). If you want to test it with Everything, it should run with no problems.

**Comment by resure** [comment]
Thanks!

Unfortunately, it fails with `everything` too:
1. Open i18n search key using everyting plugin popup

**Atom Version**: 1.0.0
**System**: Mac OS X 10.10.3
**Thrown From**: [rails-i18n](https://github.com/mauricioszabo/atom-rails-i18n) package, v0.1.2

### Stack Trace

Uncaught TypeError: Cannot read property 'append' of undefined

```
At /Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:19

TypeError: Cannot read property 'append' of undefined
  at Finder.initialize (/Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:19:23)
  at Finder.View (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at Finder.SelectListView (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:18:51)
  at new Finder (/Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:10:3)
  at atom-workspace.<anonymous> (/Users/resure/.atom/packages/rails-i18n/lib/rails-i18n.coffee:44:11)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at Object.function (/Users/resure/.atom/packages/everything/lib/commands-provider.coffee:25:21)
  at EverythingView.confirmed (/Users/resure/.atom/packages/everything/lib/everything-view.coffee:60:5)
  at EverythingView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:241:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:180:20)

```

### Commands

```
     -0:35.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:35.7.0 rails-i18n:search-key (atom-pane.pane.active)
     -0:27.5.0 command-palette:toggle (atom-pane.pane.active)
     -0:25.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:25.8.0 everything:fuzzy-finder (atom-pane.pane.active)
     -0:24.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:23.2.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
  2x -0:23.2.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:09.4.0 command-palette:toggle (atom-pane.pane.active)
     -0:05.8.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
  2x -0:05.8.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:04.3.0 everything:fuzzy-finder (atom-pane.pane.active)
     -0:02.6.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:01.2.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:00.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.9.0 rails-i18n:search-key (atom-pane.pane.active)
```

### Config

``` json
{
  "core": {
    "disabledPackages": [
      "bemhtml",
      "language-yate"
    ]
  }
}
```

### Installed Packages

``` coffee
# User
.bin, vundefined
Stylus, v1.0.0
atom-typescript, v4.6.9
everything, v0.2.0
linter, v1.0.8
rails-i18n, v0.1.2
vim-mode, v0.54.0

# Dev
No dev packages
```

**Comment by mauricioszabo** [comment]
Ok, it should be correct now (it was using some outdated APIs that, for some reason, I didn't saw on Deprecation Cop) - fixed on 0.1.3 version.

The problem you had when using Everything is because the documentation was confused - it's not suposed for you to run the "Search Keys" commands in everything, you just need to type your key or translation and Everything will try to match it. I updated README to show this usage information too.

Could you check if it's really corrected?

**Comment by resure** [comment]
Everything is fine now, thank you.
